ts/demoTodo.ts

```diff
@@ -0,0 +1,17 @@
+type TypeTodo = [
+  {
+    name: string;
+    id: number;
+    completed: boolean;
+  }
+];
+
+const demoTodo: TypeTodo = [
```

Comment:
By the way, it's not demo todo. It's demo **todos**. Your demo todos are a tuple of 1 todo. 

You can declare you demoTodo with a `satisfies` operator, so it keeps it's defined structure but also supports type checking.